tests/test_fixtures.py

```diff
@@ -45,7 +45,7 @@ class TestTemplate2(Template):
     def package(self):
         mapping = {'name': 'package.name',
                    'version': 'package.version',
-                   'license': 'package.license',
+                   'pkg_license': 'package.pkg_license',
```

Comment:
Same here - would be great if `package.pkg_license` was replaced with `package.license`